README.md

```diff
@@ -86,8 +86,8 @@ The codebase contains unit tests and integration tests.
 
 Run all tests with `poetry run pytest`, or run individual tests with `poetry run pytest path/to/test_file`.
 
-## Heroku
+## Azure
 
-This codebase is deployed to Heroku at [http://framasw-todo-app.herokuapp.com/](http://framasw-todo-app.herokuapp.com/).
+This codebase is deployed to Azure at [https://framasw-todo-app.azurewebsites.net/](https://framasw-todo-app.azurewebsites.net/).
```

Comment:
Definitely not important, but a nice opportunity to mention that [if the link text matches the link itself, you can just use `<link>`](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses):
```suggestion
This codebase is deployed to Azure at <https://framasw-todo-app.azurewebsites.net/>.
```